lib/api-call.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const axios = require('axios');
+
+module.exports = {
+  getResponses: function(baseUrl, surveyID, endPoint, apiToken, apiTokenSecret) {
+    axios.get(baseUrl + '/' + surveyID + '/' + endPoint + '?api_token=' + apiToken + '&api_token_secret=' + apiTokenSecret)
```

Comment:
good idea @chrisns 
yeah it was backtick but my sublime editor was complaining about es6 syntax.  

👍 @daniel-ac-martin